src/api/app/views/webui/user/_involvement.html.haml

```diff
@@ -1,60 +1,91 @@
-.card.mb-3#involvement
```

Comment:
This view now contains the code from *src/api/app/views/webui/user/user_profile_redesign/_involvement.html.haml*.